plugins/modules/packaging/os/pkgng.py

```diff
@@ -432,7 +450,9 @@ def main():
     # Operate on named packages
     named_packages = [pkg for pkg in pkgs if pkg != '*']
     if p["state"] in ("present", "latest") and named_packages:
-        _changed, _msg, _out, _err = install_packages(module, pkgng_path, named_packages, p["cached"], p["pkgsite"], dir_arg, p["state"])
+        _changed, _msg, _out, _err = install_packages(module, pkgng_path, named_packages, \
```

Comment:
That shouldn't be needed:
```suggestion
        _changed, _msg, _out, _err = install_packages(module, pkgng_path, named_packages,
```
(same for the next line)